Restore telemetry

fbricon opened the pull request:
### What does this PR do?
Restore telemetry collection without the need for vscode-commons. See #515.

### What issues does this PR fix or reference?
Fixes #458 for good

### Is it tested? How?
When debugging the yaml extension, check telemetry events in:
- host debug console
- https://app.segment.com/redhat-devtools/sources/vs_code_tests/debugger
 
<!-- Please provide instructions here how reviewer can test your changes if applicable -->

evidolob commented:
@fbricon Could you remove `package-lock.json` file, as we now using `yarn` and that file is redundant.

fbricon commented:
> @fbricon Could you remove `package-lock.json` file, as we now using `yarn` and that file is redundant.

My bad, old habits. I removed it and updated yarn.lock instead

joshuawilson commented:
@evidolob you could add package.lock to the .gitignore so it doesn't get accidentally committed.
